software/base/src/main/java/org/apache/brooklyn/entity/software/base/AbstractSoftwareProcessSshDriver.java

```diff
@@ -310,6 +310,20 @@ public void runPostInstallCommand() {
     }
 
     @Override
+    public void runPreCustomizeCommand() {
+        if(Strings.isNonBlank(getEntity().getConfig(BrooklynConfigKeys.PRE_CUSTOMIZE_COMMAND))) {
+            execute(getEntity().getConfig(BrooklynConfigKeys.PRE_CUSTOMIZE_COMMAND), "running pre-customize commands");
+        }
+    }
+
+    @Override
+    public void runPostCustomizeCommand() {
+        if (Strings.isNonBlank(entity.getConfig(BrooklynConfigKeys.POST_CUSTOMIZE_COMMAND))) {
+            execute(ImmutableList.of(entity.getConfig(BrooklynConfigKeys.POST_CUSTOMIZE_COMMAND)), "running post-customize commands");
```

Comment:
I'd copied the pre- and post- install command methods, and hadn't noticed that one is using a list and one wasn't. As the pre- and post- launch commands both use a list, I'll update this (and pre-install) to use a list